app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizard.js

```diff
@@ -42,22 +53,48 @@ class PlanWizard extends React.Component {
       hideAlertAction();
     }
 
-    if (activeStepIndex === 2) {
-      const plansBody = createMigrationPlans(planWizardGeneralStep, planWizardVMStep);
-
-      setPlanScheduleAction(planWizardOptionsStep.values.migration_plan_choice_radio);
+    if (
+      activeStepIndex === 2 &&
+      ((planWizardAdvancedOptionsStep.values.preMigrationPlaybook &&
+        planWizardAdvancedOptionsStep.values.playbookVms.preMigration.length === 0) ||
+        (planWizardAdvancedOptionsStep.values.postMigrationPlaybook &&
+          planWizardAdvancedOptionsStep.values.playbookVms.postMigration.length === 0))
+    ) {
+      const onConfirm = () => {
+        this.setState({ activeStepIndex: 3 });
+        hideConfirmModalAction();
+      };
+
+      showConfirmModalAction({
+        title: __('No VMs Selected'),
+        body: __(
+          "You've selected a pre-migration or post-migration playbook service but no VMs on which to run the playbook service. Are you sure you want to continue?"
+        ),
```

Comment:
Please make sure that the entire gettext text appears in a single.
And then mark that line with `// prettier-ignore`

Reply:
whoops!